ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -37,6 +37,8 @@
 
 AddV2 = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.math.add))
 
+Atan2 = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.math.atan2))
```

Comment:
Need to specify unsupported dtypes as `tf.raw_ops.Atan2` Doesn't support `float16` for examples You can use the `with_unsupported_dtypes` decorator

Reply:
Added that, thank you.